chaynHQ/bloom-frontend
# [WIP] About you forms

Adds `/account/about-you` page, redirected to after register. 

This page first asks demographic questions, which can optionally be skipped. The second form (accessed by completing the first) asks questions about coping mechanisms/progress in their healing journey, again can be optionally skipped. 
These form values will be sent to google sheets with a unique but anonymous ID for cross referencing future form values.

Completed so far
- added page and form components
- added new styles for fields with static (always at the top) label. This was required as the titles of the questions are long, vs existing 'email' labels etc.
- routing to redirect to this page after register, and to redirect to `?q=a` and `/courses`

To do
- add/fix state for some form values
- fix state values for user's data e.g. partners and features
- full logic to send these values to google sheets, with a unique id

To test
- go to `/account/about-you` after logging in/registering
- fill in the form
- click continue
- fill out form setA `/account/about-you?q=a` 
- check styling throughout
